pkg/network/tracer/ebpf_conntracker.go

```diff
@@ -135,6 +152,24 @@ func NewEBPFConntracker(cfg *config.Config) (netlink.Conntracker, error) {
 	return e, nil
 }
 
+func getAllMapsNames(mgr *manager.Manager) []string {
```

Comment:
There are at least two other variants of this? Can we make them call this function? Same for `getAllProbeNames`